### ISSUE TITLE
US state totals are missing from time series data files

### ISSUE THREAD

**Issue by JamesFrancisco** [CSSEGISandData/COVID-19#1667]
In prior updates to the repo. Time series data for the United States was broken out by state. In the 3/26 updates, that data has been aggregated into a single U.S. data point, This reduction in available data makes it impossible to assess state and regional differences in the spread of COVID-19. This is a critical defect for data scientists trying to quantify regional effects and needs to be addressed as soon as possible.

**Comment by contextofislands** [comment]
Agreed. We were relying on this data to provide reports on the US territories that often get ignored and now have no recourse.

**Comment by tarasinf** [comment]
Please point me to US data. If you decide to split this data: global and country data, why do you keep reporting Canada by region?

**Comment by cipriancraciun** [comment]
I have built a derived dataset based on JHU dataset, where (among other fixes and features) I also aggregate the data for US at both state and country level, and merge everything in a single dataset for easy usage:

* https://github.com/cipriancraciun/covid19-datasets
* I have described it in this issue #1281 or the readme in the repo above;

(I try as much as possible not to change the format too much, mainly I only add new rows and columns.)

**Comment by davidbau** [comment]
Dupe of #1505
